src/ucp/tag/tag_send.c

```diff
@@ -58,7 +58,8 @@ ucp_tag_send_req(ucp_request_t *req, size_t dt_count,
     size_t rndv_rma_thresh;
     size_t rndv_am_thresh;
 
-    if (param->op_attr_mask & UCP_OP_ATTR_FLAG_FAST_CMPL) {
+    if ((param->op_attr_mask & UCP_OP_ATTR_FLAG_FAST_CMPL) &&
```

Comment:
How bad is 256k rndv thresh for GPU mem? It is not optimal for CPU mem as well, bu this way we ensure faster local completion, possibly delaying remote completion.
So this change may have impact on blocking `MPI_Send` performance.

Reply:
@brminich it is significant. for example, cuda ipc latency

```
size        with PR    w/o PR
2048   | 7.91        | 6.38
4096   | 13.78       | 11.47
8192   | 23.35       | 21.28
16384  | 13.16       | 45.34
32768  | 13.95       | 90.79
65536  | 15.31       | 174.4
131072 | 18.15       | 349.31
262144 | 23.76       | 20.57
524288 | 34.39       | 31.27
```
In earlier version of the fix #5121 also, it was overriding {nbr_}_rma-threshold with new memtype_rma_thresh(8k).  I think we dont need new variable to control memtype rma threshold given that caluclated rma_threshold (~8k) is good for CUDA as well in a first step.

Reply:
ok